src/pages/Account/CodeOfConduct.tsx

```diff
@@ -40,14 +40,11 @@ export const CodeOfConduct: React.FC = () => {
 
   const { user } = useUser();
 
-  const venueId = useVenueId();
-
-  const { currentVenue, isCurrentVenueLoaded } = useConnectCurrentVenueNG(
-    venueId
-  );
+  const spaceSlug = useSpaceParams();
+  const { space, isLoaded: isSpaceLoaded } = useSpaceBySlug(spaceSlug);
 
   const { world, isLoaded: isWorldLoaded } = useCurrentWorld({
-    worldId: currentVenue?.worldId,
+    worldId: space?.worldId,
```

Comment:
Come to think of it, might be returning both `spaceId` and `worldId` from `useSpaceBySlug()` next to `space` and `isLoaded`. Search for !!!venueId!!!